metron-interface/metron-alerts/e2e/alerts-list/alerts-list.po.ts

```diff
@@ -231,10 +231,12 @@ export class MetronAlertsPage {
   }
 
   clickRemoveSearchChip() {
-    let aceLine = element.all(by.css('.ace_keyword')).get(0);
-    browser.actions().mouseMove(aceLine).perform().then(() => {
-      this.waitForElementPresence(element(by.css('.ace_value i'))).then(() => {
-        element.all(by.css('.ace_value i')).get(0).click();
+    element(by.css('app-alerts-list .ace_text-input')).sendKeys('').then(() => {
```

Comment:
I'd like to see a couple things with this. Can you add some code comments along with perhaps splitting this up into more descriptive functions? There's a lot of chaining/lambda/closures going on here.